src/Identity/ApiAuthorization.IdentityServer/src/Microsoft.AspNetCore.ApiAuthorization.IdentityServer.csproj

```diff
@@ -5,9 +5,6 @@
     <Description>ASP.NET Core API Authorization package powered by Identity Server.</Description>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;apiauth;identity</PackageTags>
-    <HasReferenceAssembly>false</HasReferenceAssembly>
-    <IsPackable>true</IsPackable>
-    <HasReferenceAssembly>false</HasReferenceAssembly>
```

Comment:
Making doubly sure!

Reply:
This is not part of the shared framework. Doesn't need a ref assembly.

Reply:
It won't get one. The default is `false` for any project that doesn't set `$(IsAspNetCoreApp)` to `true`.

Reply:
Wasn't aware of that.